wgpu-core/src/command/bundle.rs

```diff
@@ -1076,8 +1076,7 @@ pub mod bundle_ffi {
             .push(RenderCommand::SetPipeline(pipeline_id));
     }
 
-    #[no_mangle]
-    pub extern "C" fn wgpu_render_bundle_set_index_buffer(
+    pub fn wgpu_render_bundle_set_index_buffer(
```

Comment:
We should probably just make it a method on `RenderBundleEncoder` then. No need for a standalone function, it's just confusing the reader.